### PR TITLE
Multi-line aggregation did not explain date/time quirks

### PR DESCRIPTION
**Pull request by mariuszlip**
### What does this PR do? What is the motivation?
Multiline logs rendered by [NLog](https://github.com/NLog/NLog/wiki/LongDate-Layout-Renderer) are not captured even if they match pattern \d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d{4}

It turns out that DataDog does not recognise date with 4 fraction digits after 'seconds' component
Bad: `2024-10-23 11:02:07.4093`
Good: `2024-10-23 12:18:39.305`

The information about accepted date formats is "hidden" [here](https://docs.datadoghq.com/service_management/events/pipelines_and_processors/date_remapper/)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

Feel free to move this change around if you deem that it belogs to a different page, like 'Logs troubleshooting'

### CODE REVIEW

File: content/en/agent/logs/advanced_log_collection.md [hunk 1]
```diff
@@ -453,7 +453,7 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
 
-<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em></div>
+<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with procesion of up to millisecond, so logs with greater precision will not be sent even if they match the pattern.</div>
```
Comment (rtrieu) [inline content/en/agent/logs/advanced_log_collection.md:456]:
```suggestion
<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with precision of up to millisecond, so logs with greater precision are not sent even if they match the pattern.</div>
```

did you mean "procession" or "precision" here?

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback but otherwise looks good.

File: content/en/agent/logs/advanced_log_collection.md (hunk 1 above)
Comment (mariuszlip) [reply content/en/agent/logs/advanced_log_collection.md]: Yes, that was a typo, thank you

File: content/en/agent/logs/advanced_log_collection.md [hunk 2]
```diff
@@ -453,7 +453,7 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
 
-<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em></div>
+<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with precision of up to millisecond, so logs with greater precision are not sent even if they match the pattern.</div>
```
Comment (estherk15) [inline content/en/agent/logs/advanced_log_collection.md:456]:
```suggestion
<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> <br><br>Log collection works with precision of up to millisecond. Logs with greater precision are not sent even if they match the pattern.</div>
```

**Review by estherk15 — APPROVED** [review]
Thanks for making that update!

**Pull request merged**